### DISCUSSION TITLE
Questions about buying a domain on Vercel

### DISCUSSION THREAD

**Discussion by ChristianIvicevic** [vercel/community#770] [Help] [answered]
I've got some questions about buying a domain on Vercel. It lacks documentation what the offer includes.

1. Is it safe to assume it includes subdomains as well? Is there a limit?
2. Also, when I click on a domain it asks which account I want to connect it with. I guess it is not strictly tied to my personal Hobby account and easily transferable if I upgrade to a Pro account, right?
3. For my other domains from a different provider the public (!) Reg-C entries (not the legal ones) are linked to the provider keeping my personal details safe. Does Vercel do that as well?

**Comment by leerob** [comment, marked as answer]
- Yes, you can add subdomains. There isn't a limit (often used with [wildcard domains](https://vercel.com/docs/concepts/projects/custom-domains#wildcard-domains)).
- Yes, you can [move domains](https://vercel.com/docs/concepts/projects/custom-domains#moving-domains) between accounts.
- I am not sure on that one, I will need to check with the team.

**Reply by ChristianIvicevic** [reply]
fyi, it seems as if my personal data is not public and the Reg-C entry is pointing to Vercel. I pulled the trigger and purchased a domain the other day and have confirmed that.
